Updating hint box to suggest using External Login Providers on Cloud (#7058)
umbraco/UmbracoDocs: 15/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md

This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.

{% hint style="success" %}
**Note for Umbraco Cloud users** 
Umbraco Cloud now supports External Identity Providers, including Entra ID (formerly Azure AD).  

> If you're working on a Cloud project, we recommend following the Cloud-specific guidance here: [Set up External Login Providers on Umbraco Cloud](https://docs.umbraco.com/umbraco-cloud/set-up/external-login-providers).  

Reviewer: ```suggestion
If you're working on a Cloud project, see the [External Login Providers](https://docs.umbraco.com/umbraco-cloud/set-up/external-login-providers) article in the Umbraco Cloud documentation.  
```

> Configuration for Entra ID on Umbraco Cloud is handled via settings rather than custom code, so this tutorial is mainly relevant for non-Cloud (on-premise/self-hosted) projects.

Reviewer: ```suggestion
On Umbraco Cloud, Entra ID is configured via settings rather than custom code, so this tutorial is intended for non-Cloud (on-premises or self-hosted) projects.
```
